src/sinks/sematext/logs.rs

```diff
@@ -30,7 +30,7 @@ pub struct SematextLogsConfig {
 
     /// The endpoint to send data to.
     ///
-    /// Setting this option overrides the `region`.
+    /// Setting this option overrides the `region` option.
```

Comment:
Yeah with this now, I'd almost just do

"This option overrides the `region` option."